Misc changes

just-carlod opened the pull request:
See individual commit messages
fixes #336

melix99 commented on src/session/content/event_row.rs:
```diff
@@ -91,5 +91,11 @@ impl EventRow {
+        let label = self.imp().label.to_owned();
+        label.set_justify(gtk::Justification::Center);
+        label.set_wrap(true);
+        label.set_wrap_mode(pango::WrapMode::Char);
```
You should set these properties inside the [ui file](https://github.com/melix99/telegrand/blob/main/data/resources/ui/content-event-row.ui) directly.

just-carlod replied:
right

melix99 commented on data/resources/ui/sidebar-row.ui:
```diff
@@ -116,7 +117,6 @@
             <child>
               <object class="GtkLabel" id="unread_count_label">
                 <property name="valign">center</property>
-                <property name="ellipsize">end</property>
```
Since we've found out that that's not the reason of the bug you can just leave it.

melix99 commented on data/resources/ui/content-event-row.ui:
```diff
@@ -5,7 +5,11 @@
+      <object class="GtkLabel" id="label">
+        <property name="justify">center</property>
+        <property name="wrap">True</property>
+        <property name="wrap-mode">char</property>
```
```suggestion
        <property name="wrap-mode">word-char</property>
```

It's a much better wrap mode.

just-carlod committed:
sidebar-row: Fix unread messages counter not fitting into sidebar

Fixes melix99#336

just-carlod committed:
event-row: Reduce horizontal space

Make sure event row won't occupy too much horizontal space,
wrap and center the label if needed.

just-carlod committed:
sidebar-row: Shorten message text for thumbnail messages

We shouldn't display a "Photo" string if there's a photo thumbnail
and if the message has a captions. Same goes for other message types
that have a thumbnail.

just-carlod committed:
sidebar-row: Fix thumbnails not being centered vertically

Fixes #338
